project/Dependencies.scala

```diff
@@ -83,7 +83,9 @@ object Dependencies {
   lazy val mockito_all = "org.mockito"                 % "mockito-all"              % "1.9.0"   % "test"
   lazy val scalacheck  = "org.specs2"                 %% "specs2-scalacheck"        % "3.8.6"   % "test"
   lazy val specs2      = "org.specs2"                 %% "specs2-core"              % "3.8.6"   % "test"
+  lazy val specs2Main  = "org.specs2"                 %% "specs2-core"              % "3.8.6"
```

Comment:
Why did we need this main-classpath variant?

Reply:
webkit provides some things that depend on specs2 (e.g. MockWeb)